src/Components/Submitting.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useRef } from 'react'
+import { Modal } from 'react-bootstrap'
+import { connect } from 'scrivito'
+
+export const Submitting = connect(function Submitting() {
+  const dialogRef = useRef<HTMLDialogElement>(null)
+  useEffect(() => dialogRef.current?.showModal(), [])
+
+  return (
+    <>
+      <dialog className="d-none" ref={dialogRef} />
```

Comment:
FMI: Why is `showModal` etc needed?

Reply:
This is how dialog works. From https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog#open
> It is recommended to use the .show() or .showModal() method to render dialogs, rather than the open attribute. If a <dialog> is opened using the open attribute, it is non-modal.

Reply:
But isn't this the responsibility of `Modal` from `react-bootstrap`?

Reply:
1. By only using Modal I was not able to prevent clicking a link if the spinner was inside a link.
2. By only using dialog I was not able to easily get the visuals of Modal.

I think we could make it work with 2 (use only dialog and a div, get rid of Modal) if we invest in dedicated CSS.